### ISSUE TITLE
Running VOC2012

### ISSUE THREAD

**Issue by RBirkeland** [AustinDoolittle/Pytorch-Gain#1]
Hey,

I'm trying to replicate the results from the paper, by training it on the VOC2012 dataset (same as the paper), however it's not quite clear what the folder structure should look like from your help manual.

Could you give an example of what folder structure main.py is expecting?

Also does the network only accept one input size due to the argument --input-dims? 

Thank you.

**Comment by RBirkeland** [comment]
Ok think I've figured it out. It does not out of the box support the VOC2012 dataset as in the paper, but rather root/labels/images, and input-dims is not the input size but rather the size to transform the input images to.

However, there does not seem to be support for semi-supervised learning with pixel level masks as described in the paper. I might try to implement that myself in the future.

**Comment by AustinDoolittle** [comment]
Sorry it took so long to get back to you, I stopped short of implementing the semi supervised segmentation as described in the paper. It is definitely something I intend to implement in the near future, but contributions are more than welcome!

**Comment by hchoi71** [comment]
Hello, I know this session was closed a few years ago, but do you have any updates on how to load VOC2012 dataset to train the model?
